harmony-db/.dockerignore

```diff
@@ -0,0 +1,2 @@
+node_modules
```

Comment:
Not blocking, but these probably aren't needed since they are for NodeJS and the DB doesn't use any JS

Reply:
That goes for both entries in this file